content/reference/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2017-02-20
```

Comment:
I guess we won't merge it today, tomorrow probably, 2017-02-23 makes more sense.